### PR TITLE
Update Pinterest.

### PR DESCRIPTION
**Pull request by timonus**
Pinterest now supports two factor auth, this PR updates the entry for it.

### CODE REVIEW

File: _data/social.yml [hunk 1]
```diff
@@ -158,7 +158,12 @@ websites:
       url: https://pinterest.com
       twitter: Pinterest
       img: pinterest.png
-      tfa: No
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      exceptions:
+          text: "Software tokens can only be provided through Authy at this time."
```
Comment (mxxcon) [inline _data/social.yml]: It doesn't sound like they are planning to support other totp clients. it seems like both sms and tokens are generated completely within twillo. Unless i'm misreading their announcement?

**Comment by 0x0ece** [comment]
@mxxcon correct, we use Authy for everything. Eng blog post here:
https://medium.com/@Pinterest_Engineering/two-factor-authentication-on-pinterest-238aa3dc92d1

**Comment by 0x0ece** [comment]
Also, the help page: https://help.pinterest.com/en/articles/two-factor-authentication

**Comment by mxxcon** [comment]
@timonus Please update your `doc` link with the one @0x0ece posted. That one is more useful for people to figure out how to turn on 2fa.
And I'm thinking that perhaps `exception` text is unnecessary since it doesn't seem like you are planning to support other totp clients?

File: _data/social.yml [hunk 2]
```diff
@@ -158,7 +158,12 @@ websites:
       url: https://pinterest.com
       twitter: Pinterest
```
Comment (mxxcon) [inline _data/social.yml]: twitter info is not necessary for `tfa: yes` sites.

**Review by mxxcon — CHANGES_REQUESTED** [review]

**Comment by timonus** [comment]
@mxxcon you got it, updated.

**Review by stephengroat — APPROVED** [review]
thanks for the pr @timonus! i just removed the `twitter` tag and pointed the `doc` link to the help page

File: _data/social.yml (hunk 2 above)
Comment (timonus) [reply _data/social.yml]: Looks like @stephengroat did this.

**Review by mxxcon — APPROVED** [review]

**Comment by 0x0ece** [comment]
Out of curiosity, when will it be deployed? (just waiting to do some screenshots to tweet around)

**Comment by Carlgo11** [comment]
@0x0ece either when I get back from vacation (in a week) or if @jamcat22 merges. Let’s hope on the second one :wink:

**Comment by 0x0ece** [comment]
Thanks @Carlgo11, enjoy your vacation!

**Pull request merged**